### DISCUSSION TITLE
conf=null

### DISCUSSION THREAD

**Discussion by dronespov** [ultralytics/ultralytics#3431] [Q&A]
Hello, im training a yolov8n model for image object detection. My _loss decrease but I look at conf and its =null. Does anyone know why this is occurring? Ive includes the train results.
![results](https://github.com/ultralytics/ultralytics/assets/77562708/ad012dfc-3e39-4bf0-9ea9-3cc7a420c043)

- Andrew K

**Comment by glenn-jocher** [comment]
@dronespov hello Andrew 👋,

It seems like you're encountering an issue where the confidence (conf) value is not being displayed during training. This could be due to a variety of reasons, such as a visualization bug or a problem with the logging configuration.

First, ensure that you're using the latest version of the YOLOv8 repository, as updates often fix these kinds of issues. If you're already up to date, it might be helpful to check your training configuration file for any anomalies related to the logging of metrics.

If the problem persists, please open an issue on the YOLOv8 GitHub repository with detailed information about your training setup and the steps to reproduce the issue. This will help us diagnose the problem more effectively.

For further guidance on training and troubleshooting, you can refer to our documentation at https://docs.ultralytics.com.
